libraries/bootstrap.php

```diff
@@ -41,6 +41,12 @@ class_exists('\\Joomla\\CMS\\Autoload\\ClassLoader');
 // Decorate Composer autoloader
 spl_autoload_register([new \Joomla\CMS\Autoload\ClassLoader($loader), 'loadClass'], true, true);
 
+// Check if load of the environment variables is enabled.
+if (is_file(JPATH_ROOT . '/.env') || is_file(JPATH_ROOT . '/.env.development')) {
+    $dotenv  = Dotenv\Dotenv::createImmutable(JPATH_ROOT, ['.env', '.env.development']);
+    $dotenv->safeLoad();
```

Comment:
```suggestion
    \Dotenv\Dotenv::createImmutable(JPATH_ROOT, ['.env', '.env.development'])->safeLoad();
```

I think we don't need the global `$dotenv` variable.